### PR TITLE
Update README.md and CONTRIBUTING.md and more files

### LINKED ISSUE #3335: Update README.md, CONTRIBUTING.md, and SECURITY.md
### Provide a description of requested docs changes

See #3334.

### PR DESCRIPTION
**Pull request by skanda890**
## Summary of the pull request

Update README.md, CONTRIBUTING.md, and SECURITY.md to remove mistakes.

## References and relevant issues

#3335

## Detailed description of the pull request / Additional comments

Update README.md, CONTRIBUTING.md, and SECURITY.md to remove mistakes

## PR checklist

- [X] Closes #3335
- [X] Documentation updated
- [ ] Tests added/passed
- [ ] Telemetry [compliance tasks](https://aka.ms/devhome-telemetry) completed for added/updated events

### CODE REVIEW

**Comment by skanda890** [comment]
If I did something wrong please tell me.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,13 +28,13 @@ Therefore, if you do file issues or create PRs, please keep an eye on your GitHu
 
 ## Before you start, file an issue
 
-Please follow this simple rule to help us eliminate any unnecessary wasted effort & frustration, and ensure an efficient and effective use of everyone's time - yours, ours, and other community members':
+Please follow this simple rule to help us eliminate any unnecessary wasted effort & frustration, and ensure efficient and effective use of everyone's time - yours, ours, and other community members:
```
Comment (krschau) [inline CONTRIBUTING.md:31]: Please revert both changes in this line. The apostrophe signifies ownership and was correct.

File: README.md [hunk 2]
```diff
@@ -98,15 +98,15 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
 
 ## Code of conduct
 
-We welcome contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+We welcome contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and do, actually grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
```
Comment (krschau) [inline README.md:101]: Please revert.

File: SECURITY.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ If you believe you have found a security vulnerability in any Microsoft-owned re
 
 Instead, please report them to the Microsoft Security Response Center (MSRC) at [https://msrc.microsoft.com/create-report](https://aka.ms/security.md/msrc/create-report).
 
-If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/security.md/msrc/pgp).
+If you prefer to submit without logging in, send an email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/security.md/msrc/pgp).
```
Comment (krschau) [inline SECURITY.md:15]: Please revert.

Comment (skanda890) [reply SECURITY.md:15]: But my changes are correct.

File: SECURITY.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Instead, please report them to the Microsoft Security Response Center (MSRC) at
 
 If you prefer to submit without logging in, send email to [secure@microsoft.com](mailto:secure@microsoft.com).  If possible, encrypt your message with our PGP key; please download it from the [Microsoft Security Response Center PGP Key page](https://aka.ms/security.md/msrc/pgp).
 
-You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://www.microsoft.com/msrc). 
+Just so you know, you should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. Additional information can be found at [microsoft.com/msrc](https://www.microsoft.com/msrc).
```
Comment (krschau) [inline SECURITY.md:17]: Please do not make changes to the Security.md file.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -6,9 +6,9 @@ Below is our guidance for how to report issues, propose new features, and submit
...
 
-The point of doing all this work in public is to ensure that we are holding ourselves to a high degree of transparency, and so that the community sees that we apply the same processes and hold ourselves to the same quality-bar as we do to community-submitted issues and PRs. We also want to make sure that we expose our team culture and "tribal knowledge" that is inherent in any closely-knit team, which often contains considerable value to those new to the project who are trying to figure out "why the heck does this thing look/work like this???"
+The point of doing all this work in public is to ensure that we are holding ourselves to a high degree of transparency, and so that the community sees that we apply the same processes and hold ourselves to the same quality bar as we do to community-submitted issues and PRs. We also want to make sure that we expose our team culture and "tribal knowledge" that is inherent in any closely-knit team, which often contains considerable value to those new to the project who are trying to figure out "why the heck does this thing look/work like this???"
```
Comment (krschau) [inline CONTRIBUTING.md:11]: Please put the hyphen back.

File: README.md [hunk 6]
```diff
@@ -98,15 +98,15 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
 
 ## Code of conduct
 
-We welcome contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+We'd like to hear from you about contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and do, actually grant us the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
```
Comment (krschau) [inline README.md:101]: Please change this paragraph back.

File: README.md [hunk 7]
```diff
@@ -98,15 +98,15 @@ If you would like to ask a question that you feel doesn't warrant an issue (yet)
...
 
 ## Trademarks
 
-This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft trademarks or logos is subject to and must follow [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general). Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship. Any use of third-party trademarks or logos is subject to those third-parties' policies.
+This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft trademarks or logos is subject to and must follow [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general). Use of Microsoft trademarks or logos in modified versions of this project must not confuse or imply Microsoft sponsorship. Any use of third-party trademarks or logos is subject to those third parties' policies.
```
Comment (krschau) [inline README.md:109]: Please change this paragraph back.

**Review by krschau — CHANGES_REQUESTED** [review]

**Review by krschau — APPROVED** [review]

**Comment by skanda890** [comment]
See #3429

**Comment by skanda890** [comment]
Since you approved changes, please merge this.

**Review by EricJohnson327 — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ This is our preferred method.
 
 For users who are unable to install Dev Home from the Microsoft Store, released builds can be manually downloaded from this repository's [Releases page](https://github.com/microsoft/devhome/releases).
 
-#### Via Windows Package Manager CLI (aka winget)
+#### Via Windows Package Manager CLI (aka Winget)
```
Comment (cinnamon-msft) [inline README.md:34]:
```suggestion
#### Via Windows Package Manager CLI (aka WinGet)
```

File: README.md [hunk 9]
```diff
@@ -57,7 +57,7 @@ The Dev Home dashboard displays Windows widgets. These widgets are built using t
 
 ### Machine configuration
 
-The machine configuration tool utilizes the Dev Home GitHub Extension, but isn't required to clone and install apps. The app installation tool is powered by winget.
+The machine configuration tool utilizes the Dev Home GitHub Extension but isn't required to clone and install apps. The app installation tool is powered by Winget.
```
Comment (cinnamon-msft) [inline README.md:60]:
```suggestion
The machine configuration tool utilizes the Dev Home GitHub Extension but isn't required to clone and install apps. The app installation tool is powered by WinGet.
```

File: .github/ISSUE_TEMPLATE/Bug_Report.yml [hunk 10]
```diff
@@ -1,5 +1,5 @@
 name: "Report a bug 🐛"
-description: Report errors or unexpected behavior
+description: Report errors or unexpected behavior.
```
Comment (krschau) [inline .github/ISSUE_TEMPLATE/Bug_Report.yml:2]: Please revert.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -45,13 +45,13 @@ If no existing item describes your issue/feature, great - please file a new issu
...
 * Don't understand how to do something? File an issue
-* Found an existing issue that describes yours? Great - upvote and add additional commentary / info / repro-steps / etc.
+* Found an existing issue that describes yours? Great - upvote and add additional commentary/info/repro-steps/etc.
 
 When you hit "New Issue", select the type of issue closest to what you want to report/ask/request.
 
 ### Complete the template
 
-**Complete the information requested in the issue template, providing as much information as possible**. The more information you provide, the more likely your issue/ask will be understood and implemented. Helpful information includes:
+**Complete the information requested in the issue template, providing as much information as possible**. The more information you provide, the more likely your issue / ask will be understood and implemented. Helpful information includes:
```
Comment (krschau) [inline CONTRIBUTING.md:54]: Please revert.

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -106,53 +106,53 @@ Generally, we categorize issues in the following way, which is largely derived f
...
 
 We will not accept or review PRs that add new features to Windows Customization without an associated issue that follows the `Windows Customization` template.
 
-For bug fixes, please still use the existing [bug template](https://github.com/microsoft/devhome/issues/new?template=Bug_Report.yml). If you are able to fix the bug, please indicate that in the issue and we'll be happy to review your PR.
+For bug fixes, please still use the existing [bug template](https://github.com/microsoft/devhome/issues/new?template=Bug_Report.yml). If you can fix the bug, please indicate that in the issue and we'll be happy to review your PR.
```
Comment (krschau) [inline CONTRIBUTING.md:121]: Please revert.

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -106,53 +106,53 @@ Generally, we categorize issues in the following way, which is largely derived f
...
 
-Small issues that do not require a spec will be labelled `Issue-Bug` or `Issue-Task`.
+Small issues that do not require a spec will be labeled `Issue-Bug` or `Issue-Task`.
 
 However, some issues/features will require careful thought & formal design before implementation. For these scenarios, we'll request that a spec is written and the associated issue will be labeled `Issue-Feature`.
 
-Specs help collaborators discuss different approaches to solve a problem, describe how the feature will behave, how the feature will impact the user, what happens if something goes wrong, etc. Driving towards agreement in a spec, before any code is written, often results in simpler code, and less wasted effort in the long run.
+Specs help collaborators discuss different approaches to solving a problem, describe how the feature will behave, how the feature will impact the user, what happens if something goes wrong, etc. Driving towards agreement in a spec, before any code is written, often results in simpler code, and less wasted effort in the long run.
```
Comment (krschau) [inline CONTRIBUTING.md:131]: Please revert.

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -106,53 +106,53 @@ Generally, we categorize issues in the following way, which is largely derived f
...
 
-Specs are written in markdown, stored under the [`\docs\specs`](./docs/specs) folder and named `[issue id] - [spec description].md`.
+Specs are written in markdown, stored under the [`\docs\specs`](./docs/specs) folder, and named `[issue id] - [spec description].md`.
 
 👉 **It is important to follow the spec templates and complete the requested information**. The available spec templates will help ensure that specs contain the minimum information & decisions necessary to permit development to begin. In particular, specs require you to confirm that you've already discussed the issue/idea with the team in an issue and that you provide the issue ID for reference.
 
 Team members will be happy to help review specs and guide them to completion.
 
-### Help wanted
+### Help Wanted
```
Comment (krschau) [inline CONTRIBUTING.md:145]: Please revert. As I've mentioned before, we use "sentence case" for headers, not title case.

Comment (skanda890) [reply CONTRIBUTING.md:145]: Oh okay.

**Comment by skanda890** [comment]
@krschau, changes done.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -4,19 +4,19 @@ Below is our guidance for how to report issues, propose new features, and submit
 
 ## Open development workflow
 
-The Dev Home team is VERY active in this GitHub Repo. In fact, we live in it all day long and carry out all our development in the open!
+The Dev Home team is VERY active in this GitHub Repo. In fact, we live in it all day and carry out all our development in the open!
```
Comment (krschau) [inline CONTRIBUTING.md:7]: Please stop making additional changes. Please revert this.

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -4,19 +4,19 @@ Below is our guidance for how to report issues, propose new features, and submit
...
 
-When the team finds issues, we file them in the repo. When we propose new ideas or think-up new features, we file new feature requests. When we work on fixes or features, we create branches and work on those improvements. And when PRs are reviewed, we review in public - including all the good, the bad, and the ugly parts.
+When the team finds issues, we file them in the repo. We file new feature requests when we propose new ideas or think up new features. When we work on fixes or features, we create branches and work on those improvements. And when PRs are reviewed, we review them in public - including all the good, the bad, and the ugly parts.
```
Comment (krschau) [inline CONTRIBUTING.md:9]: Please revert this.

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -4,19 +4,19 @@ Below is our guidance for how to report issues, propose new features, and submit
...
 
 ### Repo bot
 
 The team triages new issues several times a week. During triage, the team uses labels to categorize, manage, and drive the project workflow.
 
 We employ a bot engine to help us automate common processes within our workflow.
 
-We drive the bot by tagging issues with specific labels which cause the bot engine to close issues, merge branches, etc. This bot engine helps us keep the repo clean by automating the process of notifying appropriate parties if/when information/follow-up is needed, and closing stale issues/PRs after reminders have remained unanswered for several days.
+We drive the bot by tagging issues with specific labels which cause the bot engine to close issues, merge branches, etc. This bot engine helps us keep the repo clean by automating the process of notifying appropriate parties if / when information / follow-up is needed, and closing stale issues / PRs after reminders have remained unanswered for several days.
```
Comment (krschau) [inline CONTRIBUTING.md:19]: Please revert this.

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -95,22 +95,22 @@ If you don't have any additional info/context to add but would like to indicate
...
 
 Generally, we categorize issues in the following way, which is largely derived from our old internal work tracking system:
-* ["Bugs"](https://github.com/microsoft/devhome/issues?q=is%3Aopen+is%3Aissue+label%3A%22Issue-Bug%22+) are parts of Dev Home that are not quite working the right way. There's code to already support some scenario, but it's not quite working right. Fixing these is generally a matter of debugging the broken functionality and fixing the wrong code.
+* ["Bugs"](https://github.com/microsoft/devhome/issues?q=is%3Aopen+is%3Aissue+label%3A%22Issue-Bug%22+) are parts of Dev Home that are not quite working the right way. There's code to already support some scenarios, but it's not quite working right. Fixing these is generally a matter of debugging the broken functionality and fixing the wrong code.
```
Comment (krschau) [inline CONTRIBUTING.md:105]: Please revert this.

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -150,9 +150,9 @@ Once the team has approved an issue/spec, development can proceed. If no develop
 
 ## Development
 
-### Fork, clone, branch and create your PR
+### Fork, clone, branch, and create your PR
 
-Once you've discussed your proposed feature/fix/etc. with a team member and you've agreed on an approach or a spec has been written and approved, it's time to start development:
+Once you've discussed your proposed feature/fix/etc. With a team member and you've agreed on an approach or a spec has been written and approved, it's time to start development:
```
Comment (krschau) [inline CONTRIBUTING.md:155]: Please revert this. The period does not designate the end of a sentence.

File: CONTRIBUTING.md [hunk 20]
```diff
@@ -95,22 +95,23 @@ If you don't have any additional info/context to add but would like to indicate
...
+## Contributing fixes/features
 
 If you're able & willing to help fix issues and/or implement features, we'd love your contribution!
 
 The best place to start is the list of ["good first issue"](https://github.com/microsoft/devhome/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22++label%3A%22good+first+issue%22+)s. These are bugs or tasks that we on the team believe would be easier to implement for someone without any prior experience in the codebase. Once you're feeling more comfortable in the codebase, feel free to just use the ["Help Wanted"](https://github.com/microsoft/devhome/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22+) label, or just find an issue you're interested in and hop in!
 
 Generally, we categorize issues in the following way, which is largely derived from our old internal work tracking system:
 * ["Bugs"](https://github.com/microsoft/devhome/issues?q=is%3Aopen+is%3Aissue+label%3A%22Issue-Bug%22+) are parts of Dev Home that are not quite working the right way. There's code to already support some scenario, but it's not quite working right. Fixing these is generally a matter of debugging the broken functionality and fixing the wrong code.
+
```
Comment (krschau) [inline CONTRIBUTING.md:106]: Please remove this extra line

**Comment by skanda890** [comment]
@krschau, done.

**Comment by krschau** [comment]
There should be 2 files changed in this PR, there are currently 654. I would suggest trying to rebase your branch on the current main branch. I think that being 440 commits behind is making this much more difficult for you.

**Pull request closed without merging**